cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -80,7 +79,7 @@ Then("First/Last {int} FlatTable rows {word} visible", (count, state) => {
 });
 
 Then("I click on {int} body row", (index) => {
-  flatTableBodyRowByPositionInIframe(index).click();
+  flatTableBodyRowByPosition(index).click({ force: true });
```

Comment:
Why does it have to be a forced click?

Reply:
without `iFrame` it is overlapped with other `DOM` element - that's why we need to force click